APM/apm_init_mod.F

```diff
@@ -132,7 +132,12 @@ MODULE APM_INIT_MOD
       INTEGER :: id_BCPO,id_BCPI,id_OCPO, id_OCPI
       INTEGER :: id_MSA
       INTEGER :: id_SALA, id_SALC
-      INTEGER :: id_DST1,id_DST2,id_DST3,id_DST4
+
+      ! NOTE: Renamed these to GC_DSTbin{1..7} to avoid clashing
+      ! with the id_DSTbin1 species flag (@yantosca, 31 Oct 2025)
```

Comment:
With version control do we need to put name and date next to these sorts of changes?